lib/validate.ex

```diff
@@ -70,4 +70,16 @@ defmodule Fields.Validate do
 
     Regex.match?(regex, url)
   end
+
+  def ip_address(ip_address) do
+    # https://stackoverflow.com/questions/53497/regular-expression-that-matches-valid-ipv6-addresses
+    {:ok, regex_ipv6} =
+      Regex.compile(
+        "(([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,5}(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[0-9a-fA-F]{1,4}){1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4}|([0-9a-fA-F]{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((:[0-9a-fA-F]{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4}){0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9])\.){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))"
+      )
+
+    # https://stackoverflow.com/questions/5284147/validating-ipv4-addresses-with-regexp
+    {:ok, regex_ipv4} = Regex.compile("((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\.|$)){4}")
+    Regex.match?(regex_ipv6, ip_address) || Regex.match?(regex_ipv4, ip_address)
```

Comment:
Nice and simple checker. 👍 
I would consider putting the `regex_ipv4` _first_ because it's still used by 68% of ISPs:
https://www.google.com/intl/en/ipv6/statistics.html
![image](https://user-images.githubusercontent.com/194400/82748839-e6bfa600-9d9c-11ea-98db-5966c7a3e20d.png)

If current growth trends continue it will take another 3 years before `IPv6` is used by the majority of people.
So running the `regex_ipv4` will mean fewer CPU cycles for anyone running `Fields`. 💭 
(_Happy to leave it the way it is as the cycles are probably negligible ... but I would be interested in a Perf benchmark to know for sure!_)  🐎